sources/PPU/Background.hpp

```diff
@@ -85,20 +94,46 @@ namespace ComSquare::PPU
 		void setBpp(int bpp);
 		//! @brief setter for private variable _tileMaps
 		//! @param tileMaps The tileMaps to set
-		void setTilemaps(Vector2<bool> tileMaps);
+		void setTileMapMirroring(Vector2<bool> tileMaps);
 
 		//! @brief Get the BackGround Number
 		//! @return the current Background number
-		int getBgNumber() const;
-		//! @brief set the Background priority
-		//! @param bgNumber the new Background priority
-		void setPriority(bool priority);
-		//! @brief Get the Background priority
-		//! @return the current Background priority
-		bool getPriority() const;
+		[[nodiscard]] int getBgNumber() const;
+
+
+		//! @brief Add a bg buffer to another buffer
+		template <int levelLow, int levelHigh, std::size_t DEST_SIZE_X, std::size_t DEST_SIZE_Y>
+		static void mergeBackgroundBuffer(std::array<std::array<uint32_t, DEST_SIZE_Y>, DEST_SIZE_X> &bufferDest,
+		                                  std::array<std::array<unsigned char, DEST_SIZE_Y>, DEST_SIZE_X> &pixelDestinationLevelMap,
+		                                  const Background &backgroundSrc
+		                                  )
+		{
+			int i = 0;
```

Comment:
Use real variable names, we understand that they are iterators but nothing more.

Reply:
They're nothing than iterator 
other names would be pixel and line